connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -56,7 +56,7 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
     {
       maxRequestsPerCrawl: MAX_PAGES,
       maxConcurrency: CONCURRENCY,
-
+      maxRequestsPerMinute: 300, // 5 requests per second to avoid overloading the target website
```

Comment:
I tiiink we can decrease this a bit that seems like aaa lot!

How about 60?

Reply:
Good for me too.